packages/myst-cli/src/process/myst.ts

```diff
@@ -49,7 +50,7 @@ export function parseMyst(
     extensions: {
       frontmatter: !opts?.ignoreFrontmatter,
     },
-    roles: [reactiveRole, ...(session.plugins?.roles ?? [])],
+    roles: [reactiveRole, iconRole, ...(session.plugins?.roles ?? [])],
```

Comment:
Happy with a merge except this, which we should coordinate with a theme release as that may change behaviour of the package.